.github/workflows/prerelease.yml

```diff
@@ -19,7 +19,7 @@ jobs:
         uses: actions/checkout@v4.1.1
 
       - name: Install pnpm
-        run: npm install -g pnpm@7.30.1
+        run: npm install -g pnpm
```

Comment:
Does removing the `@<version number>` automatically install the `latest` tag? If not, should we be doing `@latest`?

Reply:
Yes, it'll install the latest version if no tag/version number is provided. Here is the official link: https://pnpm.io/installation#using-npm